google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/SourceSnippets.java

```diff
@@ -32,9 +32,13 @@
 import com.google.protobuf.FieldMask;
 import java.io.IOException;
 
-/** Snippets for how to work with Sources in Cloud Security Command Center. */
+/**
+ * Snippets for how to work with Sources in Cloud Security Command Center.
```

Comment:
"Cloud Security Command Center" is old. Should just be "Security Command Center"

Reply:
Updated it across files. Thanks.